master/static/srv/get_logs_offset_limit.sql

```diff
@@ -0,0 +1,12 @@
+WITH logs AS (
```

Comment:
non-blocking: This can be simplified to just be the inner `SELECT`, since the two `ORDER BY`s are the same.